src/js/pages/locations/app.js

```diff
@@ -65,6 +65,11 @@ const initFindHelp = function (currentLocation) {
     postcode: document.querySelector('.js-find-help-postcode')
   }
 
+  const allOrganisationsOption = document.createElement('option')
+  allOrganisationsOption.setAttribute('value', 'all-service-providers')
+  allOrganisationsOption.innerText = 'All Organisations'
+  ui.select.appendChild(allOrganisationsOption)
+
```

Comment:
where is the test around this?

Reply:
I couldn't find any test for testing functionality from this file. I tried to write a test for this piece of code but I didn't have access to the 'document' object (I've got error ReferenceError: document is not defined)

Reply:
document is the browser, it needs to be wrapped and then stubbed in the test. have a look at browser.js (i think)

Reply:
ok, just seen that none of the rest is tested either. no worries